website/content/docs/operations/health.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: Boundary Health Endpoints
+description: |-
+  Verify the Boundary controller server is up and able to receive requests
+---
+
+## Boundary Health Endpoints
+
+Boundary provides health monitoring through the `/health` path using a listener
+with the `"ops"` purpose. By default, a listener with that purpose runs on port
+`9203`. See the [example configuration](#example-configuration) section for an
+example listener stanza in a `config.hcl` file.
+
+### Requirements
+
+To enable the controller health endpoint, any Boundary instance must be
+started with a controller. That is, a `controller` block and a `purpose = "api"`
+listener must be defined in Boundary's configuration file. Additionally, a
+`purpose = "ops"` listener must also be defined in Boundary's configuration
+file. Under these conditions, the `ops` server (which hosts the controller health
+api) will be exposed.
+
+### Shutdown Grace Period
```

Comment:
For the first sentence, my suggestion is to break it out into multiple sentences. Here's my suggestion, but please feel free to nix if my attempt to provide more clarity changed the intent of the sentence. Also, we'd want to avoid starting out a sentence with "Optionally."

**Suggestion**:
When the controller health endpoint is enabled, Boundary may be configured to change the controller health response to `503 Service Unavailable`. The controller health response will only change upon receiving a shutdown signal. An amount of time may be configured before the shutdown process begins.